plugins/search/api-report.md

```diff
@@ -66,7 +66,6 @@ export type FiltersState = {
 //
 // @public (undocumented)
 export const HomePageSearchBar: ({
-  className: defaultClassName,
```

Comment:
Breaking change here. I think it's fine to make, but best call it out in the changeset and making a `minor` bump

Reply:
Thnx, updated changeset with instructions too!